x-pack/legacy/plugins/graph/public/angular/templates/_sidebar.scss

```diff
@@ -49,6 +49,10 @@
   margin-bottom: 0;
 }
 
+.gphSelectionList__icon {
```

Comment:
As the circle in the sidebar selection list is much smaller, the icon should also be smaller, otherwise the icon overlaps the circle in some cases.